src/commands/backslash.rs

```diff
@@ -588,6 +595,15 @@ pub async fn execute(cmd: &BackslashCmd, prompt: &mut repl::State)
                         prompt.print.max_items = Some(limit);
                     }
                 }
+                IdleTransactionTimeout(t) => {
+                    let secs = t.value.expect("only set here");
+                    prompt.idle_transaction_timeout = if secs == 0 {
```

Comment:
Not sure why we're doing that `0 => None` dance if `0` on the server also means no timeout. It's rather confusing that `idle_transactio_timeout = None` could mean "default" or could mean "no timeout".